hw1/task2.js

```diff
@@ -0,0 +1,18 @@
+const fs = require("fs");
+const csv = require("csvtojson");
+
+const CONSTANTS = require("./constants");
+
+const writeOutput = fs.createWriteStream("output.txt");
+const source = fs

```

Comment:
you don't need source variable, just call fs

Reply:
Fixed!